openpype/hosts/resolve/hooks/pre_resolve_setup.py

```diff
@@ -41,61 +46,64 @@ def execute(self):
                 "/Applications/DaVinci Resolve/DaVinci Resolve.app"
                 "/Contents/Libraries/Fusion/fusionscript.so"
             ),
-            "linux": "/opt/resolve/libs/Fusion/fusionscript.so"
+            "linux": "/opt/resolve/libs/Fusion/fusionscript.so",
         }
-        RESOLVE_SCRIPT_LIB = os.path.normpath(
-            RESOLVE_SCRIPT_LIB_[current_platform])
-        self.launch_context.env["RESOLVE_SCRIPT_LIB"] = RESOLVE_SCRIPT_LIB
+        resolve_script_lib = Path(resolve_script_lib_dirs[current_platform])
+        self.launch_context.env[
+            "RESOLVE_SCRIPT_LIB"
+        ] = resolve_script_lib.as_posix()
 
-        # TODO: add OTIO installation from  `openpype/requirements.py`
+        # TODO: add OTIO installation from `openpype/requirements.py`
         # making sure python <3.9.* is installed at provided path
-        python3_home = os.path.normpath(
-            self.launch_context.env.get("RESOLVE_PYTHON3_HOME", ""))
+        python3_home = Path(
+            self.launch_context.env.get("RESOLVE_PYTHON3_HOME", "")
+        )
 
-        assert os.path.isdir(python3_home), (
+        assert python3_home.is_dir(), (
             "Python 3 is not installed at the provided folder path. Either "
             "make sure the `environments\resolve.json` is having correctly "
             "set `RESOLVE_PYTHON3_HOME` or make sure Python 3 is installed "
             f"in given path. \nRESOLVE_PYTHON3_HOME: `{python3_home}`"
         )
-        self.launch_context.env["PYTHONHOME"] = python3_home
-        self.log.info(f"Path to Resolve Python folder: `{python3_home}`...")
+        python3_home_str = python3_home.as_posix()
+        self.launch_context.env["PYTHONHOME"] = python3_home_str
+        self.log.info(f"Path to Resolve Python folder: `{python3_home_str}`")
 
-        # add to the python path to path
+        # add to the python path to PATH
         env_path = self.launch_context.env["PATH"]
-        self.launch_context.env["PATH"] = os.pathsep.join([
-            python3_home,
-            os.path.join(python3_home, "Scripts")
-        ] + env_path.split(os.pathsep))
+        self.launch_context.env[
+            "PATH"
+        ] = f"{python3_home_str}{os.pathsep}{env_path}"
```

Comment:
Again, `Scripts` folder exists only on Windows python installation, and it works without this path anyways.

Reply:
Actually I'm not sure why would we need to add python path to the PATH at all, both for Fusion and Resolve. It does not seem to have any sense. For instance, on macos there's no executables on that path. Everything is in the `bin` folder, including `python3`. On Windows python executable is in the PYTHONHOME, but all other executables is in Scripts. 

Resolve scripting reference also does not require any PATH modifications:

    You may need to set the these environment variables to allow for your Python installation to pick up the appropriate dependencies as shown below:

    Mac OS X:
    RESOLVE_SCRIPT_API="/Library/Application Support/Blackmagic Design/DaVinci Resolve/Developer/Scripting"
    RESOLVE_SCRIPT_LIB="/Applications/DaVinci Resolve/DaVinci Resolve.app/Contents/Libraries/Fusion/fusionscript.so"
    PYTHONPATH="$PYTHONPATH:$RESOLVE_SCRIPT_API/Modules/"

    Windows:
    RESOLVE_SCRIPT_API="%PROGRAMDATA%\Blackmagic Design\DaVinci Resolve\Support\Developer\Scripting"
    RESOLVE_SCRIPT_LIB="C:\Program Files\Blackmagic Design\DaVinci Resolve\fusionscript.dll"
    PYTHONPATH="%PYTHONPATH%;%RESOLVE_SCRIPT_API%\Modules\"

    Linux:
    RESOLVE_SCRIPT_API="/opt/resolve/Developer/Scripting"
    RESOLVE_SCRIPT_LIB="/opt/resolve/libs/Fusion/fusionscript.so"
    PYTHONPATH="$PYTHONPATH:$RESOLVE_SCRIPT_API/Modules/"
    (Note: For standard ISO Linux installations, the path above may need to be modified to refer to /home/resolve instead of /opt/resolve)